shared/src/schemas/Session.ts

```diff
@@ -40,6 +40,7 @@ const LiveSessionFieldsSchema = yup
     url: yup.string().required(),
     link: yup.string(),
     inviteCode: yup.number().required(),
+    hostingCode: yup.number(),
```

Comment:
Before merging this make sure this is not included in any session responses

Reply:
Where does the field selection/exclusion happens?
It is taking some time to be able to find it in the code, doesn't seem to be in neither the modal, controller or route 🤔